plugins/woocommerce/src/Blocks/BlockTemplatesController.php

```diff
@@ -26,43 +18,23 @@ class BlockTemplatesController {
 	 */
 	const TEMPLATES_ROOT_DIR = 'templates';
 
-	/**
-	 * Package instance.
-	 *
-	 * @var Package
-	 */
-	private $package;
-
 	/**
 	 * Constructor.
-	 *
-	 * @param Package $package An instance of Package.
 	 */
-	public function __construct( Package $package ) {
-		$this->package = $package;
-
-		$feature_gating                                 = $package->feature();
-		$is_block_templates_controller_refactor_enabled = $feature_gating->is_block_templates_controller_refactor_enabled();
-
-		// This feature is gated for WooCommerce versions 6.0.0 and above.
-		if ( defined( 'WC_VERSION' ) && version_compare( WC_VERSION, '6.0.0', '>=' ) && ! $is_block_templates_controller_refactor_enabled ) {
-			$this->init();
-		}
+	public function __construct() {
+		$this->init();
```

Comment:
Same as `BlockTemplateRegistry`, I think we should invoke this method in the Bootstrap class instead of inside the constructor.

Reply:
Same, done in 713c43afdbbc067c0e639e5022afa84925d982bf.